### PR TITLE
Andrew/bugfix

### PR DESCRIPTION
**Pull request by andrewkalumba**
review my updates

### CODE REVIEW

File: html/project-page.html [hunk 1]
```diff
@@ -24,43 +24,58 @@ <h1 class="header__page-title">
...
+                                <path 
+                                    d="M16.4 4c0-1.326-1.074-2.4-2.4-2.4s-2.4 1.074-2.4 2.4c0 0.961 0.568 1.784 1.384 2.167-0.082 1.584-1.27 2.122-3.335 2.896-0.87 0.327-1.829 0.689-2.649 1.234v-4.121c0.825-0.38 1.4-1.209 1.4-2.176 0-1.326-1.074-2.4-2.4-2.4s-2.4 1.074-2.4 2.4c0 0.967 0.576 1.796 1.4 2.176v7.649c-0.825 0.379-1.4 1.208-1.4 2.175 0 1.326 1.074 2.4 2.4 2.4s2.4-1.075 2.4-2.4c0-0.961-0.568-1.784-1.384-2.167 0.082-1.583 1.271-2.122 3.335-2.896 2.030-0.762 4.541-1.711 4.64-4.756 0.829-0.378 1.409-1.21 1.409-2.181zM6 2.615c0.764 0 1.384 0.619 1.384 1.385 0 0.764-0.62 1.385-1.384 1.385s-1.384-0.621-1.384-1.385c0-0.766 0.619-1.385 1.384-1.385zM6 17.385c-0.766 0-1.385-0.621-1.385-1.385 0-0.766 0.619-1.385 1.385-1.385 0.764 0 1.384 0.619 1.384 1.385 0 0.764-0.62 1.385-1.384 1.385zM14 5.385c-0.766 0-1.385-0.621-1.385-1.385 0-0.766 0.619-1.385 1.385-1.385 0.764 0 1.384 0.619 1.384 1.385 0 0.764-0.62 1.385-1.384 1.385z">
+                                </path>
+                            </svg> 
                             <span>BRANCH</span>
```
Comment (VaishaliGovindaraj) [inline html/project-page.html:65]: change it to Branches

**Review by Svpriyaa2808 — APPROVED** [review]
great to go

File: html/project-page.html (hunk 1 above)
Comment (TobiasKjernell) [inline html/project-page.html:65]: typo: Branches

**Review by TobiasKjernell — APPROVED** [review]
looks good

**Review by VaishaliGovindaraj — APPROVED** [review]
Looks good :)

**Review by ranjithavelusamy — APPROVED** [review]
its looks good

**Pull request merged**